cli/modules/modules_test.go

```diff
@@ -1,3 +1,6 @@
+// FIXME: Create new tests https://github.com/tarantool/tt/issues/1039
```

Comment:
Why not create new tests now?

Reply:
At the moment, we don't have any public method for unit tests. And no completed functionality for integration tests.